compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_loops_for_vectorization.mlir

```diff
@@ -9,7 +9,8 @@
 !t4_f32  = tensor<64x64x64x64xf32>
 
 
-module {
+#executable_target_amdaie_xclbin_fb = #hal.executable.target<"amd-aie", "amdaie-xclbin-fb", {target_device = "npu1_4col", ukernels = "none"}>
```

Comment:
It's useful to add a test without device as well and check that the device not found error is thrown.